packages/neon_lints/lib/src/base.yaml

```diff
@@ -0,0 +1,227 @@
+analyzer:
+  language:
+    strict-casts: true
+  errors:
+    flutter_style_todos: ignore
+    todo: ignore
```

Comment:
I think we should eventually enable these. Leaving instead of leaving todos in the code we should just create issues. Of course you can still use todos during development and then the CI will also be able to warn you about it.